resources/views/admin/passwords/reset.blade.php

```diff
@@ -0,0 +1,65 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Reset Password') }}</div>
+
+                <div class="card-body">
+                    <form method="POST" action="{{ route('password.update') }}">
```

Comment:
formタグについても学ぶ必要がありますね！

```suggestion
                    <form method="POST" action="{{ route(`user.password.update`) }}">
```
ですかね！

Reply:
https://webukatu.com/wordpress/blog/33869/#GETPOST:~:text=%E3%81%A6%E3%81%84%E3%81%8D%E3%81%BE%E3%81%99%E3%80%82-,GET%E3%81%A8POST%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6,-GET%E3%81%A8POST

Reply:
@issiyda 
すみません！こちらはadminの方でまだ修正しておりませんでした！
userの方では{{ route(`user.password.update`) }}としているのですが、表示されない状況です、、